# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/fake/dvb-apps/disable-ttusb-dec-reset.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./util/Makefile.orig	2004-12-17 20:24:46.181958488 +0100
+++ ./util/Makefile	2004-12-17 20:24:53.519842960 +0100
@@ -8,6 +8,5 @@
 	$(MAKE) -C dvbdate $(MAKECMDGOALS)
 	$(MAKE) -C dvbtraffic $(MAKECMDGOALS)
 	$(MAKE) -C av7110_loadkeys $(MAKECMDGOALS)
-	$(MAKE) -C ttusb_dec_reset $(MAKECMDGOALS)
 
 FORCE:
